tests/test_generation_scripts.py

```diff
@@ -1,39 +0,0 @@
-# Copyright (c) Microsoft Corporation.
```

Comment:
Removed since we used this purely for code coverage tracking (https://github.com/microsoft/playwright-python/pull/99) and code coverage tracking got removed since a while.